include/dev/emmc.h

```diff
@@ -0,0 +1,288 @@
+                                uint32_t arg2, emmc_resp_t *resp) {
+  device_t *idev = EMMC_METHOD_PROVIDER(dev, send_cmd);
+  return emmc_methods(idev->parent)->send_cmd(dev, cmd, arg1, arg2, resp);
+}
+
+/**
+ * \brief Wait until e.MMC signals
+ * \param dev e.MMC controller device
+ * \param wflags conditions to be met
+ * \return 0 on success, ETIMEDOUT on timeout
+ */
+static inline int emmc_wait(device_t *dev, emmc_wait_flags_t wflags) {
+  device_t *idev = EMMC_METHOD_PROVIDER(dev, wait);
+  return emmc_methods(idev->parent)->wait(dev, wflags);
+}
+
+/**
+ * \brief Read data fetched by the controller
+ * \param dev e.MMC controller device
+ * \param buf pointer to where the data should be written to
+ * \param len expected data length
+ * \param n pointer for the number of read bytes or NULL
+ * \return 0 on success, ENODATA if no new data can be read, EBUSY if device is
+ * busy
+ */
+static inline int emmc_read(device_t *dev, void *buf, size_t len, size_t *n) {
+  device_t *idev = EMMC_METHOD_PROVIDER(dev, read);
+  return emmc_methods(idev->parent)->read(dev, buf, len, n);
+}
+
+/**
+ * \brief Write data to the controller
+ * \param dev e.MMC controller device
+ * \param buf pointer to where the data should be read from
+ * \param len data length
+ * \param n pointer for the number of written bytes in or NULL
+ * \return 0 on success
+ */
+static inline int emmc_write(device_t *dev, const void *buf, size_t len,
+                             size_t *n) {
+  device_t *idev = EMMC_METHOD_PROVIDER(dev, write);
+  return emmc_methods(idev->parent)->write(dev, buf, len, n);
+}
+
+/**
+ * \brief Gets a value describing the e.MMC device
+ * \param dev e.MMC controller device
+ * \param id value identifier
+ * \param var pointer to where the associated value should be written to
+ * \return 0 if value was fetched successfully, non-zero if it wasn't
+ */
+static inline int emmc_get_prop(device_t *dev, emmc_prop_id_t id,
+                                emmc_prop_val_t *val) {
+  device_t *idev = EMMC_METHOD_PROVIDER(dev, get_prop);
+  return emmc_methods(idev->parent)->get_prop(dev, id, val);
+}
+
+/**
+ * \brief Sets a value describing the e.MMC device
+ * \param dev e.MMC controller device
+ * \param id value identifier
+ * \param var pointer to where the associated value should be written to
+ * \return 0 if value was fetched successfully, non-zero if it wasn't
+ */
+static inline int emmc_set_prop(device_t *dev, emmc_prop_id_t id,
+                                emmc_prop_val_t val) {
+  device_t *idev = EMMC_METHOD_PROVIDER(dev, set_prop);
+  return emmc_methods(idev->parent)->set_prop(dev, id, val);
+}
+
+static inline emmc_device_t *emmc_device_of(device_t *device) {
+  return (emmc_device_t *)((device->bus == DEV_BUS_EMMC) ? device->instance
+                                                         : NULL);
+}
+
+/* Use if a command is expected to respond with R1b */
```

Comment:
Please check comment standards if there shouldn't be sth like `\brief`

Reply:
No. This is not required.